tests/database_test_utils.py

```diff
@@ -0,0 +1,113 @@
+        :return: None
+        """
+        if specification["table"] == "task":
+            status_specification = {"id": specification["status_id"], "table": "status"}
+            if not self.item_exists(status_specification):
+                self.init([status_specification])
+
+    def get_items(self, specification: dict[str, str | int]) -> Any:
+        """
+        Create a query based on the dictionary specification and return the result
+        :param specification: a dictionary with a table specification
+        :return: the results of the query
+        """
+        values = ", ".join(
+            key + "=" + str(val) if str(val).isnumeric() else str('"' + val + '"')  # type: ignore
+            for key, val in specification.items()  # type: ignore
+            if key != "table"  # type: ignore
+        )
+        table = specification["table"]
+        statement = f"SELECT * FROM {table} WHERE {values}"  # noqa S608
+        with Session(get_engine()) as session:
+            return session.exec(text(statement)).all()  # type: ignore
+
+    def item_exists(self, specification: dict[str, Any]) -> bool:
+        """
+        Check if an item exists in the database with the table and id given
+        in the dictionary
+        :param specification: a dictionary with a table specification
+        :return: True if the item exists in the database, False otherwise
+        """
+        result = self.get_items(specification)
+        return len(result) != 0
+
+    def init(self, specifications: list[dict[str, str | int]] | None = None) -> None:
+        """
+        Given an array of specifications, create the database entries.
+
+        Example: [{'table': 'task', 'id': 1 'title': 'Test task 1', 'description': 'Test task description 1'}]
+
+        The example below will be enriched so all required fields for a task will have a value.
+
+        Example: [{'table': 'task', 'id': 1}]
+
+        Example: [{"table": "status", "id": 1},{"table": "task", "id": 1, "status_id": 1}]
+
+        :param specifications: an array of dictionaries with table specifications
+        :return: None
+        """
+        if specifications is None:
+            return
+        for specification in specifications:
+            specification = self._enrich_with_default_values(specification)
+            exists_specification = {"table": specification["table"], "id": specification["id"]}
+            if not self.item_exists(exists_specification):
+                self._fix_missing_relations(specification)
+                table = specification.pop("table")
+                keys = ", ".join(key for key in specification)
+                values = ", ".join(
+                    str(val) if str(val).isnumeric() else str("'" + val + "'")  # type: ignore
+                    for val in specification.values()  # type: ignore
+                )
+                statement = f"INSERT INTO {table} ({keys}) VALUES ({values})"  # noqa S608
```

Comment:
Why not creat an Task item or a Status Item and add it with sqlmodel?

Reply:
Did think about that, but this made things more flexible for me. Maybe it will turn out as a bad idea, we'll see.